lib/releaselib.sh

```diff
@@ -947,7 +947,7 @@ release::docker::release () {
   fi
   
   # Always reset back to $USER
-  logrun $GCLOUD config set account $USER_AT_DOMAIN
+  logrun $GCLOUD config set account ${USER_AT_DOMAIN:-$USER@$DOMAIN_NAME}
```

Comment:
Would it be prudent to quote this, like: `"${USER_AT_DOMAIN:-${USER}@${DOMAIN_NAME}}"`.. gcloud is expecting exactly one value argument

Reply:
Done.